samples/brokers/http/kubernetes/http.grpc.broker.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: akri.sh/v0
+kind: Configuration
+metadata:
+  name: http-grpc-broker-rust
+spec:
+  protocol:
+    http:
+      discoveryEndpoint: http://discovery:9999
+  capacity: 1
+  brokerPodSpec:
+    imagePullSecrets: # GitHub Container Registry secret
+      - name: ghcr
+    containers:
+      - name: http-grpc-broker-rust
+        image: ghcr.io/dazwilkin/http-grpc-broker-rust@sha256:a4a7494aef44b49bd08f371add41db917553391ea397c60e9b4d213545b94f4e
```

Comment:
perhaps the image line should be replaced with something like `image: <HTTP-GRPC-IMAGE>` and the docs should have a `sed` command that replaces it with whatever `build.sh` creates?

Reply:
Fixed (previously): https://github.com/deislabs/akri/commit/b64e52eead3ac1d67b8797b67c195490972d888e